variants/views.py

```diff
@@ -4441,6 +4441,53 @@ def post(self, *args, **kwargs):
         )
 
 
+class VariantCarriersView(PluginContextMixin, TemplateView):
+    """Render HTML with other carriers of the variant."""
+
+    template_name = "variants/filter_result/variant_carriers_result.html"
+
+    def get_context_data(self, *args, **kwargs):
+        result = super().get_context_data(*args, **kwargs)
+
+        small_vars = SmallVariant.objects.filter(
```

Comment:
Is there an index for that?

Reply:
```
            models.Index(
                fields=["case_id", "release", "chromosome", "start", "reference", "alternative"],
                name="variants_sm_case_id_coords",
            ),
```

in models.py and as must be in the migrations (because of the partitioned table)

```
            CREATE INDEX variants_sm_case_id_coords ON variants_smallvariant (case_id, release, chromosome, start, reference, alternative);
```

Reply:
In your query is no case, so the index does not apply, I assume.

Reply:
`case_id` is used for partitioning so the index should apply (it needs to go over all table partitions anyway). On our production server:

```
 Aggregate  (cost=476.60..476.61 rows=1 width=8) (actual time=572.346..572.765 rows=1 loops=1)
   ->  Append  (cost=0.42..474.04 rows=1024 width=0) (actual time=199.319..572.752 rows=3 loops=1)
         ->  Index Only Scan using variants_smallvariant_0_release_chromosome_start_reference__idx on variants_smallvariant_0  (cost=0.42..0.45 rows=1 width=0) (actual time=1.300..1.300 rows=0 loops=1)
               Index Cond: ((release = 'GRCh37'::text) AND (chromosome = '3'::text) AND (start = 126730873) AND (reference = 'G'::text) AND (alternative = 'A'::text))
               Heap Fetches: 0
[...]
```

Reply:
I think I created an index manually for this on production to speed up the helpdesk requests. This might confuse things. But if you are sure about that, I will be quiet.

Reply:
variants_sm_variant_no_case2 | CREATE INDEX variants_sm_variant_no_case2 ON ONLY public.variants_smallvariant USING btree (release, chromosome, start, reference, alternative)

Reply:
Please have a look at the production database. I do not think that you can create an index on a partitioned table without `case_id`, at least there used to be such limitations.

Reply:
Ah, there actually is a partitioned index...

```
 public | variants_sm_variant_no_case2                                    | partitioned index | varfish | variants_smallvariant
```

However that one is not used.

Reply:
cf. #494

Reply:
I hope you are right and we do not run into performance issues :pray: